app/src/features/apiClient/types.ts

```diff
@@ -68,6 +68,7 @@ export namespace RQAPI {
       preRequest: string;
       postResponse: string;
     };
+    errorMessage?: string;
```

Comment:
This shouldn't be here as it is only required for pre-script and post-script. It is not related to entity